### DISCUSSION TITLE
Error while importing deepmd

### DISCUSSION THREAD

**Discussion by alib-l** [deepmodeling/deepmd-kit#799] [Q&A]
Hello, 
I want to use deepmd with ASE but I get this error:

NotFoundError: /usr/local/lib/python3.7/dist-packages/deepmd/libop_abi.so: undefined symbol: _ZN10tensorflow11GetNodeAttrERKNS_9AttrSliceEN4absl14lts_2020_02_2511string_viewEPSt6vectorIiSaIiEE

How can I solve this?

**Comment by njzjz** [comment]
See #796.
